pytorch_sphinx_theme/layout.html

```diff
@@ -76,18 +76,13 @@
   {%- endblock %}
 
   {%- block extrahead %}
-  <!-- Google Analytics -->
-  {% if theme_analytics_id %}
-    <script async src="https://www.googletagmanager.com/gtag/js?id={{ theme_analytics_id }}"></script>
-    <script>
-      window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
-
-      gtag('config', '{{ theme_analytics_id }}');
-    </script>
-  {% endif %}
-  <!-- End Google Analytics -->
+  <!-- Google Tag Manager -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-T8XT4PS');</script>
```

Comment:
We might want to not harcode the id here and use `{{ theme_analytics_id }}` instead. Otherwise, all users using the theme would be using the same ID. By using `{{ theme_analytics_id }}` we make sure that only those repos who have set this variable in the `conf.py` will be logged https://github.com/pytorch/vision/blob/8324c481dd4c3096697332d76fbdc9d912f7360b/docs/source/conf.py#L143

```suggestion
    })(window,document,'script','dataLayer','{{ theme_analytics_id }}');</script>
```